02_configure_host.sh

```diff
@@ -63,14 +63,14 @@ else
       if [ ! -e /etc/sysconfig/network-scripts/ifcfg-provisioning ] ; then
           echo -e "DEVICE=provisioning\nTYPE=Bridge\nONBOOT=yes\nNM_CONTROLLED=no\nBOOTPROTO=static\nIPADDR=172.22.0.1\nNETMASK=255.255.255.0" | sudo dd of=/etc/sysconfig/network-scripts/ifcfg-provisioning
       fi
-      sudo ifdown provisioning || true
-      sudo ifup provisioning
+      sudo ip link set provisioning down || true
+      sudo ip link set provisioning up || true
```

Comment:
Why is this changed?

Reply:
same question about the other places where `ifdown`/`ifup` were changed to `ip link set up` / `down`

Reply:
ifdown/ifup doesn't work without net-tools package in Ubuntu, so and just change use ip. It work fine on all linux distributions, and we don't need to install additional packages for it.

Reply:
I don't think those commands are equivalent, though?

Reply:
Yeah, but in context of dev-env, I don't see any problems when we use `ip link set`. 
Do you think are there any problems with it ? But If you really think that this change is not problematic, I will drop it in this pull request.

Reply:
I'd at least rather see it in a separate PR since it doesn't seem related to adding kinder support